app/crud/assets.py

```diff
@@ -37,7 +37,7 @@ async def get_assets(dataset: str, version: str) -> List[ORMAsset]:
 
     return rows
 
-@alru_cache(maxsize=128)
+
 async def get_raster_tile_sets():
```

Comment:
Can this function actually be removed entirely now? I don't see any other usages after the changes to routes/datasets/queries.py